### PR TITLE
Release 1.1.8

### PR DESCRIPTION
**Pull request by thiemowmde**
Please don't forget to merge #79!

… and possibly #70, please?

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -93,6 +93,11 @@ It is based upon and contains a lot of code written by [Jeroen De Dauw]
 
 ## Release notes
 
+### 1.1.8 (2016-10-11)
```
Comment (JeroenDeDauw) [inline README.md]: Well...

**Comment by JeroenDeDauw** [comment]
+2... not merging yet as I'm not sure if you want 79 merged before or not

**Comment by thiemowmde** [comment]
The order does not matter, but I believe #79 is crucial for this release.

File: README.md (hunk 1 above)
Comment (mariushoch) [inline README.md]: Updated.

**Pull request merged**